packages/babel-jest/src/index.ts

```diff
@@ -22,12 +23,18 @@ const THIS_FILE = fs.readFileSync(__filename);
 const jestPresetPath = require.resolve('babel-preset-jest');
 const babelIstanbulPlugin = require.resolve('babel-plugin-istanbul');
 
+// Narrow down the types
```

Comment:
quite happy with how this turned out. By using an interface, we can narrow down the type, and it'll tell us here if we break `Transform`'s contract, and not just a local one.

And importers of `babel-jest` directly will see that `createTransformer` is always there - it's not possibly undefined. Pretty nice!